### ISSUE TITLE
Plain text archives

### ISSUE THREAD

**Issue by evaryont** [danluu/post-mortems#17]
We have no idea how long the linked webpages will last, let alone the entire sites themselves. Or perhaps there will be a new belief that companies should hide these mistakes and they'll retroactively delete them (or public access anyways).

There should be a plain text copy (or close-enough, like the scraped HTML) stored in the repo as well.

**Comment by danluu** [comment]
Yeah, that's a good point. This has been a problem in a related repo I have, danluu/debugging-stories#3. I don't know what the copyright implications of that are. I think I'm not allowed to just have a copy of some copyrighted thing? I don't know anything about copyright, though.

Some solutions that don't require having a bunch of random maybe copyrighted material here are using pinboard's archival solution, somehow making sure this stuff gets into archive.org, or just having a script here that lets anyone who's cloned the repo just run a script and grab everything. I think wget with the right options will do that, so that's not even that much work. I pretty swamped with other stuff at the moment but I'll probably do the script thing when I have a bit of spare time, unless you want to do it or have a simpler suggestion.
